Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM openjdk:8
+FROM adoptopenjdk:8
```

Comment:
do we want to use a digest instead of a tag so that the base image doesn't change out from under us again in the future?

Reply:
Not sure - It's nice getting the latest in this case as they might fix bugs / improve security?  Did @dinvlad  have anything to say about this?

Reply:
Not in our earlier discussion.  @dinvlad any guidance on this question?

Reply:
Yes, @gbggrant this was my recommendation! As long as it doesn't break your build or is easy to amend the Dockerfile to unbreak it, please use this image instead - it has much fewer findings.

Reply:
(any image or tag under `adoptopenjdk` umbrella is also fine - feel free to use any from https://hub.docker.com/_/adoptopenjdk or https://hub.docker.com/u/adoptopenjdk)

Reply:
That being said, getting the latest (`8` in this case) tag is great because you're assured of the up-to-date fixes.